GeneratorInterface/Core/src/BaseHadronizer.cc

```diff
@@ -104,7 +104,6 @@ const std::vector<std::string> BaseHadronizer::theSharedResources;
     edm::FileInPath script("GeneratorInterface/LHEInterface/data/run_generic_tarball_cvmfs.sh");
     const char *outfilename = "cmsgrid_final.lhe";
     
-    char *args[5];
```

Comment:
I think a c++ way that avoids newing char*s for this is just

std::array<std::string,4>argStrs;
argStrs[0]=script.fullPath();
argStrs[1]=gridpackPath();
argStrs[2]=nevStream.str();
argStrs[3]=randomStream.str();
std::array<char*,5>args = { &argStrs[0][0], &argStrs[1][0],&argStrs[2][0],&argStrs[3][0], NULL};

then memory is managed for you